build.gradle

```diff
@@ -119,6 +123,16 @@ subprojects { project ->
     errorproneJavac 'com.google.errorprone:javac:9+181-r4173-1'
     errorprone 'com.google.errorprone:error_prone_core:2.3.3'
   }
+
+  apply plugin: 'org.jetbrains.dokka'
+  dokka {
```

Comment:
FYI, this may not work with the `gradle-maven-publish-plugin` in the sense that it may just redeclare its own configuration that overwrites it. Could be wrong though, but I was having issues around that in a separate project

Reply:
thanks for the heads up